### PR TITLE
Adding AnnotationList transcriptions in ProjectServlet, closes #692

### LINKED ISSUE #692: New servlet to resolve AnnotationList URIs

### PR DESCRIPTION
**Pull request by markskroba**

### CODE REVIEW

File: src/java/edu/slu/tpen/servlet/ProjectServlet.java [hunk 1]
```diff
@@ -91,65 +101,87 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
...
-                                }
-                                catch(Exception e){
-                                
-                                }
-                                resp.setHeader("Access-Control-Allow-Headers", "*");
-                                resp.setHeader("Access-Control-Expose-Headers", "*"); //Headers are restricted, unless you explicitly expose them.  Darn Browsers.
-                                resp.setHeader("Cache-Control", "no-cache, no-store, must-revalidate"); // HTTP 1.1.
-                                resp.setHeader("Pragma", "no-cache"); // HTTP 1.0.
-                                resp.setHeader("Expires", "0"); // Proxies.
+                    String[] urlData = req.getPathInfo().substring(1).split("/");
```
Comment (cubap) [inline src/java/edu/slu/tpen/servlet/ProjectServlet.java:104]: This is technically nicer if you split on the expected pattern "project/" and "manifest/" but the servlet mapping should be maintaining this so it isn't required. You will only find odd behavior with something like "../manifest/3065/193881" which is wrong.

File: src/java/edu/slu/tpen/servlet/ProjectServlet.java [hunk 2]
```diff
@@ -91,65 +101,87 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
...
+                    switch (urlData.length) {
+                        case 1: 
+                            // project/* - Project
+                            projID = parseInt(req.getPathInfo().substring(1).replace("/", "").replace("manifest.json",""));
+                            Project proj = new Project(projID);
+                            //System.out.println("Project 3");
+                            if (proj.getProjectID() > 0) {
```
Comment (cubap) [inline src/java/edu/slu/tpen/servlet/ProjectServlet.java:111]: better as a guard clause

File: src/java/edu/slu/tpen/servlet/ProjectServlet.java [hunk 3]
```diff
@@ -91,65 +101,87 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
...
+                                //System.out.println("Project 4");
+                                if (new Group(proj.getGroupID()).isMember(uid) || skip) {
```
Comment (cubap) [inline src/java/edu/slu/tpen/servlet/ProjectServlet.java:113]: better as a guard clause

**Review by cubap — APPROVED** [review]
looks okay. We'll test

**Review by thehabes — APPROVED** [review]
Tested and confirmed functional.

**Pull request merged**